src/Components/ProvisioningWizard/steps/FinishProgress/index.js

```diff
@@ -128,11 +140,14 @@ const FinishStep = ({ imageID, setLaunchSuccess }) => {
               <br />
               {polledReservation?.status}
               <span className="status-error">
+                {isSessionTimeout && 'Session timeout'}
```

Comment:
I'd try to include bit more info, so users know what really is happening.
```suggestion
                {isSessionTimeout && 'Session timed out, the reservation took too long to fulfill'}
```